article/_posts/2022-06-16-What_Matters_Robot_Manipulation.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: review
+title: "What Matters in Learning from Offline Human Demonstrations for Robot Manipulation"
+tags: reinforcement
+author: "Antoine Théberge"
+cite:
+    authors: "Ajay Mandlekar, Danfei Xu, Josiah Wong, Soroush Nasiriany, Chen Wang, Rohun Kulkarni, Li Fei-Fei, Silvio Savarese, Yuke Zhu, Roberto Martín-Martín"
+    title:   "What Matters in Learning from Offline Human Demonstrations for Robot Manipulation"
+    venue:   "arXiv"
+pdf: "https://arxiv.org/pdf/2108.03298.pdf"
+---
+
+In this post I examine the content of the article from a reinforcement learning-centric point of view, but the article is focused on learning from a fixed set of demonstration in general, either via imitation learning or reinforcement learning.)
+
+# Highlights
+
+- Current offline reinforcement learning (RL) datasets were conceived from RL agents transitions
+- Learning from human demonstrations does not work with current state-of-the-art offline RL algorithms
+- Temporality helps when learning from human datasets
```

Comment:
I am not sure what you mean by temporality.

Reply:
Added some explanations in [dba308e](https://github.com/vitalab/vitalab.github.io/pull/571/commits/dba308e00370c39d8df98476ddb70865d1aa75ba)